src/inc/contract.h

```diff
@@ -46,15 +46,7 @@
 //
 //      LOADS_TYPE(level)       the function promises not to load any types beyond "level"
 //
-//      SO_INTOLERANT           the function cannot tolerate an SO at any point and must run behind an
-//                              an SO probe via BEGIN_SO_INTOLERANT_XXX.  This is the default. We want most
-//                              of our code to run behind an SO probe.  The only time you need to explicitly
-//                              mark something as SO_INTOLERANT is if the static analysis tool incorrectly
-//                              flags it as an entry point.
-//      -or- SO_TOLERANT        the function can tolerate an SO.  It either does not update any global state
-//                              that needs to be cleaned up should a random SO occur, or it protects those
-//                              updates behind an SO probe.
-//      -or- SO_NOT_MAINLINE    the function is not hardened to SO and should never run on a managed thread
+//      SO_NOT_MAINLINE         the function is not hardened to SO and should never run on a managed thread
```

Comment:
I didn't remove this one because the diff was already too big, but if SO_INTOLERANT and SO_TOLERANT is removed (which itself I am not sure of) then this one should be removed too.